### ISSUE TITLE
Restrict access

### ISSUE THREAD

**Issue by mMoliere** [majlis-erc/manuForma#258]
@wsalesky Only logged-in users should be able to access the forms.

**Comment by mMoliere** [comment]
@wsalesky This works really great! Thank you for that.

A small note: When I am inside of one form, the user name is being displayed in the upper right corner. This signals to me that I am logged in. I would also like the start page to display the user name, if I am logged in. Right now, it only displays the user icon. So, I have no idea if I am still logged in. The only way I can test this is by opening one of the forms.

Also (I don't know why didn't think of this earlier) there should be an option to log out – In case you are using manuForma from a computer that is not yours.

**Comment by wsalesky** [comment]
@mMoliere what browser are you using for testing? All those features should be there, but there may be some caching issues. I will have to investigate how to prevent the caching.

**Comment by mMoliere** [comment]
@wsalesky I have tried Chrome and Firefox. The experience is the same.

**Comment by wsalesky** [comment]
@mMoliere Thanks, I will do some more testing.

**Comment by mMoliere** [comment]
@wsalesky Yes, now can see that too! Did you change something in the meantime or did it take a while until that aspect had worked itself through the cache?
